mix.exs

```diff
@@ -42,7 +42,7 @@ defmodule Hammer.Backend.Mnesia.MixProject do
       {:credo, "~> 1.6", only: [:dev, :test]},
       {:dialyxir, "~> 1.1", only: [:dev], runtime: false},
       {:ex_doc, "~> 0.29", only: :dev},
-      {:hammer, "~> 6.1"}
+      {:hammer, github: "ruslandoga/hammer", branch: "just-use"}
```

Comment:
when we have it ready, we should make it minimum 7.x for hammer

Reply:
Yes.